package.json

```diff
@@ -36,12 +39,19 @@
     "karma-sinon": "^1.0.5",
     "phantomjs": "^2.1.7",
     "qunit": "^2.6.1",
-    "sinon": "^3.2.1"
+    "rollup": "0.68.2",
+    "rollup-plugin-commonjs": "^9.2.0",
+    "rollup-plugin-multi-entry": "^2.1.0",
+    "rollup-plugin-node-resolve": "^4.0.0",
+    "rollup-plugin-typescript": "^1.0.0",
+    "sinon": "^3.2.1",
+    "tslib": "^1.9.3",
```

Comment:
where is tslib being used?

Reply:
`tslib` is a dependency used by [rollup-plugin-typescript](https://github.com/rollup/rollup-plugin-typescript)

>Note that both typescript and tslib are peer dependencies of this plugin that need to be installed separately.